pages/paper/[documentId]/[documentSlug]/index.tsx

```diff
@@ -60,6 +65,21 @@ const DocumentIndexPage: NextPage<Args> = ({
   }) as [Paper | null, Function];
   const { revalidateDocument } = useCacheControl();
 
+  const refetchTwitterScore = async () => {
```

Comment:
Are you refetching this value here because you want to make sure we get "fresh" data? If so, I think the most consistent way to handle this would be to add it to the metadata endpoint `get_document_metadata`. I realize we chatted and you had a bunch of issues related. I can take a look if you are unable to get it to work